src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +23,27 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    private static final int MAX_LETTERS = 5;
+
+    public String getUniqueCharacters(String fileName) throws IOException {
+        StringBuilder resultStr = new StringBuilder();
+        Set<String> latterSet = new TreeSet<>();
+        BufferedReader reader = new BufferedReader(new FileReader(fileName));
+        int charValue;
+        while ((charValue = reader.read()) != -1) {
+            String symbol = String.valueOf((char) charValue).toLowerCase();
+            if (symbol.matches("[a-z]")) {
+                latterSet.add(symbol);
+            }
+        }
+        int count = 0;
+        for (String s : latterSet) {
+            count++;
+            resultStr.append(s);
+            if (count == MAX_LETTERS) {
```

Comment:
I think it may be better to extract this check to loop signature as in the end, they both return the same variable. Just think about additional checks and type of loop. Still, it is up to you :)